electron/utils.ts

```diff
@@ -37,13 +37,29 @@ async function isOnline() {
   return net.isOnline()
 }
 
-async function isEpicOffline() {
-  const epicStatusApi = 'https://status.epicgames.com/api/v2/status.json'
-  const { data } = await axios.default.get(epicStatusApi)
-  const {
-    status: { indicator }
-  } = data
-  return indicator === 'major'
+async function isEpicServiceOffline(
+  type: 'Epic Games Store' | 'Fortnite' | 'Rocket League' = 'Epic Games Store'
+) {
+  const epicStatusApi = 'https://status.epicgames.com/api/v2/components.json'
+
+  try {
+    const { data } = await axios.default.get(epicStatusApi)
+
+    for (const component of data.components) {
+      const { name: name, status: indicator } = component
+
+      // found component and checking status
+      if (name === type) {
+        return indicator === 'major'
+      }
+    }
+  } catch (error) {
+    logError(
+      `Failed to get epic service status with ${error}`,
+      LogPrefix.Backend
+    )
+    return true
```

Comment:
I wonder if something goes wrong in this request should we return that there is an error on Epic side?
I think perhaps we should return false, since other things could impact this request and the epic servers could be up.
But I am not 100% on that tbh.

Reply:
If we return false, we could remove the function completly.
The try catch is there, if the page is not reachable.

What we could do is just a frontend notification that epic services might be down?

Reply:
I mean, the return false should be on the `catch` not at the end. 
Since you are doing a try/catch, the return should be inside the block for both.

Reply:
Look again